tests/Integration/Behaviour/Features/Scenario/CatalogPriceRule/catalog_price_rules_listing.feature

```diff
@@ -1,7 +1,7 @@
-#./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags catalog-price-rule-listing
+#./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s catalog_price_rule --tags catalog-price-rule-listing
```

Comment:
```suggestion
#./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s catalog-price-rule --tags catalog-price-rule-listing
```

Reply:
Only the tag should be written in kebab-case

Reply:
hmm no, its a suit name and its snake_cased. But the tag `@catalog-price-rule` can stay (altough idk if we will ever use it)
https://github.com/PrestaShop/PrestaShop/blob/develop/tests/Integration/Behaviour/behat.yml#L243